Docs: Update prometheus.md (#30240)
grafana/grafana: docs/sources/datasources/prometheus.md

For detailed instructions, refer to [Internal Grafana metrics]({{< relref "../administration/metrics.md">}}).

## Prometheus API

The Prometheus data source works with other projects that implement the [Prometheus query API](https://prometheus.io/docs/prometheus/latest/querying/api/) including:

- Cortex
- [Thanos](https://thanos.io/v0.17/components/query.md/)

> For more information on how to get other project data into Prometheus and Grafana, refer to the specific project documentation. 

Reviewer: ```suggestion
For more information on how to query other Prometheus-compatible projects from Grafana, refer to the specific project documentation. 
```
